### ISSUE TITLE
appAppleId is required for verification in production environment but it's easy to miss

### ISSUE THREAD

**Issue by shimastripe** [apple/app-store-server-library-swift#34]
## WHY

- appAppleId is nil in readme sample code.

```swift
// try! used for example purposes only
let verifier = try! SignedDataVerifier(rootCertificates: appleRootCAs, bundleId: bundleId, appAppleId: nil, environment: environment, enableOnlineChecks: enableOnlineChecks)
```

- This code works fine in the sandbox environment, but not in the production environment.
  - Because SignedDataVerifier is checked appAppleId in production environment.
- https://github.com/apple/app-store-server-library-swift/blob/8fb466815e59a440795315303147fe91be1f0a5f/Sources/AppStoreServerLibrary/SignedDataVerifier.swift#L73-L74
- It's easy to miss, so it is better to include some measures.

## How

- Add a comment to sample code
- Print warning logs if appAppleId is nil 
- Create a function to check the production configuration

What do you think? I would be happy to receive your comments.

Thanks!

**Comment by alexanderjordanbaker** [comment]
@shimastripe
That is a good suggestion, I will plan on 
1. Adding a comment to the READMEs
2. Throwing an error in the constructor if appAppleId is null/nil/undefined and the environment is Production

**Comment by alexanderjordanbaker** [comment]
Resolved in #35
